packages/manager/apps/dedicated/client/app/billing/main/history/billing-main-history.html

```diff
@@ -136,7 +136,7 @@
                 ng-if="$row.url"
                 href="{{ :: $row.url }}"
                 external
-                on-click="$ctrl.trackInvoiceOpening()"
+                on-click="$ctrl.trackInvoiceOpening('html')"
```

Comment:
```suggestion
                data-on-click="$ctrl.trackInvoiceOpening('html')"
```

Reply:
As mentioned by @antleblanc https://github.com/ovh/manager/pull/3466#discussion_r472802183 We are shifting on this behavior. Only for webcomponents, meaning that on <oui-action-menu-item>, we are ok to stop using them.